### DISCUSSION TITLE
Output Format Description in the Documentation

### DISCUSSION THREAD

**Discussion by LukasFehring** [blue-yonder/tsfresh#974] [Q&A]
Hello Everybody,

I have a question regarding https://tsfresh.readthedocs.io/en/latest/text/data_formats.html#output-format.
The first column of the table is called 'id'. However, I believe this is not an actual table column but the df.index of the returned DataFrame df, and the documentation is therefore misleading.

Am i wrong in my assessment?

PS: This is the first time that I use something like 'discussion' on GitHub,

**Comment by nils-braun** [comment]
Hi @LukasFehring! Welcome and sorry for the massive delay in answering. 
You are correct. `id` is the index of the dataframe. Would it be less misleading to you if this is called "index"? The reason why we list it there is to make it visible to users that there will be one row per original ''id' (A, B, ...). 
If you want, you can create a small PR to fix this (or we do it).
